packages/x-components/src/main.ts

```diff
@@ -24,12 +23,12 @@ FilterEntityFactory.instance.registerModifierByFilterModelName(
 
 const installer = new XInstaller({
   ...baseInstallXOptions,
-  app: App,
-  vueOptions: {
-    router
-  },
+  rootComponent: AppComponent,
   domElement: '#app',
-  onCreateApp: initDevtools
+  onCreateApp: initDevtools,
+  async installExtraPlugins({ app }) {
```

Comment:
`async` is needed? Nothing is asynchronous inside

Reply:
Nope, it was a copy&paste 😅